<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsEditor</class>
 <widget class="QDialog" name="SettingsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>120</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <widget class="QPushButton" name="buttonStartup">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Startup</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonFavorites">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Favorites</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonMain_GUI">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Main GUI</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonDone">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>356</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>On startup:</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="startupScrollArea">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>30</y>
       <width>181</width>
       <height>321</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>179</width>
        <height>319</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonApply">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>356</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonReturn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>356</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="buttonReturn_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>356</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonApply_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>356</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Favorites:</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="favoritesScrollArea">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>30</y>
       <width>181</width>
       <height>321</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>179</width>
        <height>319</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="buttonReturn_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>356</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonApply_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>356</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Main GUI behavior</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
